RedisEnterprise-DNS-Migration/migration_script.ps1

```diff
@@ -282,28 +286,27 @@ function ExtractPrivateDnsZoneNameFromPrivateLink($privateLink) {
 
 function TagAndRemoveLinksFromOldZones($vnetToZoneMap) {
     Write-Host "Now that new zones are set up, the final step is to disconnect old zones. See step 'Testing your migration' in the migration document." -ForegroundColor Yellow 
-    Write-Host "As the old zones are disconnected they will be 'tagged' with old vnet they were attached to. This is to help with the 'Troubleshooting' section in the migration document, should it be needed." -ForegroundColor Yellow
+    Write-Host "As the old zones are disconnected they will be 'tagged' with old VNet they were attached to. The tag helps if you need to troubleshoot. For more information, see the Troubleshooting section of the migration document." -ForegroundColor Yellow
     foreach ($vnet in $vnetToZoneMap.GetEnumerator()) {
         $zones = $vnetToZoneMap[$vnet.Name]
         $oldZones = Get-PrivateDNSZonesMatchingPattern $OLD_DNS_ZONE_PATTERN $zones
         if($null -eq $oldZones) {
-            Write-Host "No old zones exist in vnet $($vnet.Name)" -ForegroundColor Yellow
+            Write-Host "No old zones exist in VNet $($vnet.Name)" -ForegroundColor Yellow
         } else {
             foreach($zone in $oldZones) {
                 
                 $links = Get-AzPrivateDnsVirtualNetworkLink -ResourceGroupName $zone.ResourceGroupName -ZoneName $zone.Name
                 foreach($link in $links) {
-                    if($link.VirtualNetworkId.ToLower() -eq $vnet.Name.toLower()) {
-                        $shouldContinue = AskForConfirmation "Deleting vnet link from $($zone.Name) to $($vnet.Name). Continue?"
+                    if($link.VirtualNetworkId.ToLower() -eq $vnet.Name.ToLower()) {
+                        $shouldContinue = AskForConfirmation "Deleting VNet link from $($zone.Name) to $($vnet.Name). Continue?"
                         if ($shouldContinue) {
                             Remove-AzPrivateDnsVirtualNetworkLink -ResourceGroupName $zone.ResourceGroupName -ZoneName $zone.Name -Name $link.Name | Out-Null
-                            Write-Host "Tagging old zone with information about the vnet is was linked to so it can be reconnected if needed" -ForegroundColor Yellow
+                            Write-Host "Tagging old zone with information about the vnet is was linked to. It can be reconnected if needed" -ForegroundColor Yellow
                             $oldTags = Get-AzTag -ResourceId $zone.ResourceId
-                            $oldValue = $oldTags.Properties.TagsProperty.OldVnetLink -split ","
-                            $oldValue = $oldValue + $vnet.Name # Append new vnet name to list
-                            $oldValue = @($oldValue | Where-Object { $_ } | Sort-Object | Select-Object -Unique) # Make sure new list values are non-empty and unique
-                            $Tags = @{"OldVnetLink" = $oldValue -join ","}
-                            New-AzTag -ResourceId $zone.ResourceId -Tag $Tags
+                            $oldTags
```

Comment:
Not that it will do any harm either way, but out of curiosity, is this original `$oldTags` variable kept here to provide additional context?